pkg/plugins/qemu/qemu.go

```diff
@@ -0,0 +1,514 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"time"
+
+	"github.com/digitalocean/go-qemu/qmp"
+	"github.com/digitalocean/go-qemu/qmp/raw"
+	"github.com/lima-vm/lima/pkg/executil"
+	"github.com/lima-vm/lima/pkg/limayaml"
+	"github.com/lima-vm/lima/pkg/plugins"
+	"github.com/lima-vm/lima/pkg/plugins/framework"
+	"github.com/lima-vm/lima/pkg/qemu"
+	"github.com/lima-vm/lima/pkg/store/filenames"
+	"github.com/sirupsen/logrus"
+)
+
+// QemuPlugin is the QEMU VM driver plugin
+type QemuPlugin struct {
+	*framework.BasePluginServer
+	instances map[string]*Instance
+}
+
+// Instance represents a running QEMU VM instance
+type Instance struct {
+	ID     string
+	Config *framework.Config
+	Status string
+	qCmd   *exec.Cmd
+	qWaitCh chan error
+	vhostCmds []*exec.Cmd
+}
+
+// NewQemuPlugin creates a new QEMU plugin
+func NewQemuPlugin() *QemuPlugin {
```

Comment:
```suggestion
func New() *Plugin {
```

"qemu" is already implied in the package name